app/controllers/categories_controller.rb

```diff
@@ -1,41 +0,0 @@
-class CategoriesController < ApplicationController
-  def index
-	@category=Category.all
```

Comment:
follow the ruby style guide throughout your code: https://github.com/bbatsov/ruby-style-guide#syntax